RevenueCatUI/CustomerCenter/ButtonStyles.swift

```diff
@@ -42,7 +42,40 @@ struct ProminentButtonStyle: PrimitiveButtonStyle {
         .applyIf(background != nil, apply: { $0.tint(background) })
         .applyIf(textColor != nil, apply: { $0.foregroundColor(textColor) })
     }
+}
 
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct CustomerCenterButtonStyle: ButtonStyle {
+    let normalColor: Color
+    let pressedColor: Color
+
+    func makeBody(configuration: ButtonStyleConfiguration) -> some View {
+        configuration.label
+            .padding(.horizontal)
+            .padding(.vertical, 12)
+            .background(configuration.isPressed ? pressedColor : normalColor)
+            .cornerRadius(10)
+    }
+}
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+extension ButtonStyle where Self == CustomerCenterButtonStyle {
+    static func customerCenterScrollView(for colorScheme: ColorScheme) -> CustomerCenterButtonStyle {
```

Comment:
I believe the name is wrong
```suggestion
    static func customerCenterButtonStyle(for colorScheme: ColorScheme) -> CustomerCenterButtonStyle {
```